packages/playground/src/components/execution/step-execution.tsx

```diff
@@ -81,8 +79,8 @@ export const StepExecution: React.FC<StageExecutionProps> = ({ step, stepIndex }
               <KeyValueTable
                 className="pt-2"
                 tableSpec={inputTable}
-                tableTitleName={step.inputTitle.name}
-                tableTitleVal={step.inputTitle.value}
+                tableTitleName={'Input Name'}
```

Comment:
Table header names should not be part of `step` details from api response/mock data.